TARGETS

```diff
@@ -411,18 +417,6 @@ ROCKS_TESTS = [['merger_test', 'table/merger_test.cc', 'serial'],
  ['stringappend_test',
   'utilities/merge_operators/string_append/stringappend_test.cc',
   'serial'],
- ['cassandra_format_test',
-  'utilities/merge_operators/cassandra/format_test.cc',
-  'serial'],
- ['cassandra_merge_test',
-  'utilities/merge_operators/cassandra/cassandra_merge_test.cc',
-  'serial'],
- ['cassandra_row_merge_test',
-  'utilities/merge_operators/cassandra/row_merge_test.cc',
-  'serial'],
- ['cassandra_serialize_test',
-  'utilities/merge_operators/cassandra/serialize_test.cc',
-  'serial'],
```

Comment:
Why are all these removed? 
I also see a few other test related changes in this diff. Is this intentional?